tools/generate.js

```diff
@@ -283,6 +293,7 @@ const sources = [
   {url: "https://github.com"},
   {url: "https://gist.github.com"},
   {url: "https://help.github.com"},
+  {url: "https://graphql-explorer.githubapp.com/"},
```

Comment:
```suggestion
  {
    url: "https://graphql-explorer.githubapp.com/", 
    prefix: ".graphql-container",
    match: [".graphql-container"]
  },
```

Reply:
@xt0rted commit @silverwind suggestion and regenerate, if it works (it should just like mobile styles go ahead and merge.